tests/integration/test_issue_2235.py

```diff
@@ -0,0 +1,67 @@
+import os
+import subprocess
+import sys
+
+import pytest
+
+from pex.compatibility import PY3
+from testing import PY38, ensure_python_interpreter
+
+
+@pytest.mark.skipif(sys.version_info < (3, 8), reason="Flask 2.3.3 requires Python3.8+.")
+def test_standard_library_is_included(
+    tmpdir,  # type: Any
+    pex_project_dir,  # type: str
+):
+    # type: (...) -> None
+
+    # N.B.: The package script requires Python 3.
+    python = sys.executable if PY3 else ensure_python_interpreter(PY38)
+
+    package_script = os.path.join(pex_project_dir, "scripts", "package.py")
+    pex_pex = os.path.join(str(tmpdir), "pex")
```

Comment:
Instead of building the Pex PEX and running that; just use testing.run_pex_command as is done here: https://github.com/pantsbuild/pex/blob/5c0c29c192bd99335b5a1ab5b68a0f8556cd97aa/tests/integration/test_issue_1936.py#L17-L18

Reply:
Thanks, this is much simpler.